modules/AWS/rds/main.tf

```diff
@@ -22,47 +22,49 @@ module "security_group" {
 }
 
 module "db" {
-  source  = "terraform-aws-modules/rds/aws"
-  version = "~> 3.0"
+  source                      = "terraform-aws-modules/rds/aws"
```

Comment:
I wonder why the indentation of this file has changed? Based on terraform linting, this kind of change should raise an issue before commit and I suspect you have not the `pre-commit` hook installed properly on your local machine. (see [pre-commit hook](https://atlassian-labs.github.io/data-center-terraform/development/HOW_TO_START/))

Reply:
I guess I changed in deliberately to have a table like representation of key-values.